btax/tests/test_params_have_effect.py

```diff
@@ -1,29 +1,42 @@
+from __future__ import unicode_literals
 import re
 
 import pytest
 
 from btax.parameters import DEFAULTS, get_params, translate_param_names
 from btax.run_btax import run_btax_to_json_tables
+import btax.front_end_util as front_end
+
+front_end.DO_ASSERTIONS = True # Override env var
+                               # Always assert table format okay
 
 def tst_once(fast_or_slow, **user_params):
     if fast_or_slow == 'slow':
         # actually run the model
         # and look at the "changed" tables
         tables = run_btax_to_json_tables(**user_params)
-        has_changed = False
-        for k in tables:
-            changed = tables[k]['changed']
-
-            for row in changed:
-                for item in row:
-                    if isinstance(item, (float, int)) and item:
-                        has_changed = True
-                        break
-                if has_changed:
-                    break
-            if has_changed:
-                break
-        assert has_changed
+        assert isinstance(tables, dict)
```

Comment:
@talumbau @jdebacker : More assertions on table format are below, checking the right number of cells per row, the right number of columns and some data types.